lib/cdo/i18n_backend.rb

```diff
@@ -64,7 +64,9 @@ module MarkdownTranslate
       def translate(locale, key, options = ::I18n::EMPTY_HASH)
         translation = super(locale, key, options)
         if options.fetch(:markdown, false)
-          @renderer ||= Redcarpet::Markdown.new(Redcarpet::Render::Safe)
+          # The safe_links_only just makes sure that the URL is a "safe" web one which starts with: "#", "/", "http://", "https://", "ftp://", "mailto:"
```

Comment:
what are the "unsafe" links that this setting is attempting to protect us from?

Reply:
I put my investigation notes in the comments. Let me know if that is good enough.